server/coordinator/procedure/storage.go

```diff
@@ -21,7 +21,7 @@ import (
 )
 
 type Write interface {
-	CreateOrUpdate(ctx context.Context, meta Meta) error
+	CreateOrUpdate(ctx context.Context, meta Meta, ttl int64) error
```

Comment:
`ttl` is ambiguous for callers, how about put it as:
```suggestion
	CreateOrUpdate(ctx context.Context, meta Meta, ttlSec int64) error
```